src/components/homepage/grantCards/GrantCard.jsx

```diff
@@ -29,14 +29,15 @@ export default function GrantCard(props) {
   const classes = useStyles();
   const [expanded, setExpanded] = React.useState(false);
   const [faved, setFaved] = React.useState(false);
-  const userId = useSelector((state) => state.login.userId);
   const handleExpandClick = () => {
     setExpanded(!expanded);
   };
 
+  const date = new Date(grant.due_date);
+  const due_date = `${date.getMonth()}/${date.getDate()}/${date.getFullYear()}`;
```

Comment:
Smart formatting approach, I'm gonna have to borrow this method in writer profile!